package.json

```diff
@@ -53,7 +52,7 @@
     "@types/apollo-upload-client": "^17.0.3",
     "@types/classnames": "^2.3.1",
     "@types/history": "^5.0.0",
-    "@types/jest": "^29.5.5",
+    "@types/jest": "^29.5.14",
```

Comment:
Isn't this be obsolete/unnecessary now that you have migrated from Jest to Vitest?

Reply:
Removed